MBBSEmu/Program.cs

```diff
@@ -177,6 +187,18 @@ private void Run(string[] args)
 
                                 break;
                             }
+                        case "-DBREBUILD":
```

Comment:
command line parsing is getting crazy. I wrote some cool args/flags parsing code for Java, perhaps I should port it....